README.rst

```diff
@@ -242,6 +242,30 @@ This script will go through the process of performing the following tasks
    * openstack configuration
    * tempest testing
 
+**Note**: When setting up a multi-node on a cloud which doesn't have access to test pulp (i.e. everywhere except SMS lab) a separate local pulp must be deployed. Before doing so, it is a good idea to make sure your seed VM has sufficient disk space by setting ``seed_disk_size`` in your ``terraform.tfvars`` to an appropriate value (100-200 GB should suffice). In order to set up the local pulp service on the seed, first obtain/generate a set of Ark credentials, then add the following configuration to ``etc/kayobe/environments/ci-multinode/stackhpc-ci.yml``
+
+.. code-block:: console
+
+   stackhpc_release_pulp_username: <ark-credentials-username>
+   stackhpc_release_pulp_password: !vault |
+          <vault-encrypted-ark-password>
+
+   pulp_username: admin
+   pulp_password: <randomly-generated-password-to-set-for-local-pulp-admin-user>
+
+You may also need to comment out many of the other config overrides in ``stackhpc-ci.yml`` such as ``stackhpc_repo_mirror_url`` plus all of the ``stackhpc_repo_*`` and ``stackhpc_docker_registry*`` variables. 
```

Comment:
Bit vague. What scenarios would mean you do/don't want to comment them out?

Reply:
If I was more familiar with the naming conventions of the test pulp vs Ark repo paths for the `stackhpc_repo_*` variables then I'd happily be more specific... Feel free to reword this bit yourself.

Reply:
I've chucked in a one line sed command to just do it all